refarch-gateway/README.md

```diff
@@ -41,17 +41,40 @@ Beside the default behaviour there are some special route prefixes which are han
-| `ALLOWED_ORIGINS_PUBLIC` (optional)                      | List of urls allowed as origin for public routes.  | `https://*.muenchen.de,http://localhost:*`                              |
-| `ALLOWED_ORIGINS_CLIENTS` (optional)                     | List of urls allowed as origin for clients routes. | `https://*.muenchen.de,http://localhost:*`                              |
-| `REFARCH_SECURITY_CSRFWHITELISTED_<index>` (optional)    | List of routes to disable csrf protection for.     | `/example/**`                                                           |
-| `INFO_APPSWITCHER_URL` (optional)                        | App switcher url for usage in refarch frontend.    | `https://appswitcher.muenchen.de`                                       |
+| Var                                                       | Description                                        | Example                                                                 |
+|-----------------------------------------------------------|----------------------------------------------------|-------------------------------------------------------------------------|
+| `SPRING_PROFILES_ACTIVE`                                  | See profiles                                       | `local,hazelcast-local`                                                 |
+| `SPRING_CLOUD_GATEWAY_ROUTES_<index>_ID`                  | Id of a route definition.                          | `backend`                                                               |
+| `SPRING_CLOUD_GATEWAY_ROUTES_<index>_URI`                 | The uri to route to if this route matches.         | `http://backend-service:8080/`                                          |
+| `SPRING_CLOUD_GATEWAY_ROUTES_<index>_PREDICATES_<index>`  | Route predicates i.e. matcher.                     | `Path=/api/backend-service/**`                                          |
+| `SPRING_CLOUD_GATEWAY_ROUTES_<index>_FILTERS_<index>`     | List of filters applied to the route.              | `RewritePath=/api/backend-service/(?<urlsegments>.*), /$\{urlsegments}` |
+| `ALLOWED_ORIGINS_PUBLIC` (optional)                       | List of urls allowed as origin for public routes.  | `https://*.muenchen.de,http://localhost:*`                              |
+| `ALLOWED_ORIGINS_CLIENTS` (optional)                      | List of urls allowed as origin for clients routes. | `https://*.muenchen.de,http://localhost:*`                              |
+| `REFARCH_SECURITY_CSRFWHITELISTED_<index>` (optional)     | List of routes to disable csrf protection for.     | `/example/**`                                                           |
+| `INFO_APPSWITCHER_URL` (optional)                         | App switcher url for usage in refarch frontend.    | `https://appswitcher.muenchen.de`                                       |
```

Comment:
Maybe provide a more general domain here as well. By the way, why is the AppSwitcher URL needed for the API gateway? Isn't the frontend directly requesting the AppSwicher-Server instance and not the API gateway?

Reply:
I don't know as i never used the appswitcher. This came from the old template.

Reply:
Okay we should investigate later on and maybe create a follow up issue to remove the AppSwitcher stuff if possible. 😄

Reply:
See #203